### PR TITLE
fix(default-theme): fix IOS problem with scrolling inside cart

### LINKED ISSUE #1665: [BUG] iOS scrolling in overlays/sidebars not working
### Project type
- [x] Project (generated by CLI)
- [ ] Contribution (cloned/forked from GH)

### Describe the bug
Scrolling seems broken on iOS (tested under 14.5) in overlays and sidebars.

Steps to reproduce:
- put multiple items in cart so that the list overflows
- try to scroll the list
- on iOS it doesn't work

### Additional context
- Safari on iOS 14.5
- We think it has something to do with the `body-scroll-lock` package, there is also a [new version ](https://github.com/willmcpo/body-scroll-lock/releases/tag/v4.0.0-beta.0) which unfortunately doesn't solve the problem

I'm trying to scroll the item list in the cart here, but it doesn't work
https://user-images.githubusercontent.com/16021919/133218397-6ba2ca40-2171-4e9a-b7b4-9f1a66c5a42c.mp4

### PR DESCRIPTION
**Pull request by patzick**
## Changes

closes #1665 

fixes the error with scrolling inside cart on IOS

### CODE REVIEW

File: packages/default-theme/src/components/SwCart.vue [hunk 1]
```diff
@@ -174,10 +175,15 @@ export default {
     }
   },
 
-  // TODO: hotfix - not final solution. To remove after fix cause of a problem.
   watch: {
     isSidebarOpen(val) {
+      this.$nextTick(() => {
```
[the remark this answers is not in the document]
Comment (patzick) [reply packages/default-theme/src/components/SwCart.vue:180]: @niklaswolf I cannot reproduce this behaviour on IOS at BrowserStack. Tried a couple of models and it looks like this:

https://user-images.githubusercontent.com/13100280/134144436-a1053936-3fef-478d-a177-00264b3c42dd.mp4

[the remark this answers is not in the document]
Comment (niklaswolf) [reply packages/default-theme/src/components/SwCart.vue:180]: @patzick did you try to swipe, not scroll using your mouse? In the video it looks like your scrolling with the mouse

[the remark this answers is not in the document]
Comment (patzick) [reply packages/default-theme/src/components/SwCart.vue:180]: @niklaswolf, yeah I did both, scrolled and dragged it. On your video, there is also a weird back button, I do not see it anywhere. Is this an actual device you're testing with?
image

[the remark this answers is not in the document]
Comment (patzick) [reply packages/default-theme/src/components/SwCart.vue]: @niklaswolf okay the only thing which worked well was to disable the body scroll lock inside mobile devices, which should fix this issue as it only appears on mobile. I hope that the library itself will implement the fix, or SFUI will do that soon. Another way is to copy SFUI component and fix it directly inside it. For now, it's all it's reasonable to do.

**Pull request merged**